packages/x-tree-view/src/internals/plugins/useTreeViewExpansion/useTreeViewExpansion.types.ts

```diff
@@ -13,10 +14,33 @@ export interface UseTreeViewExpansionPublicAPI {
 }
 
 export interface UseTreeViewExpansionInstance extends UseTreeViewExpansionPublicAPI {
-  isItemExpanded: (itemId: string) => boolean;
-  isItemExpandable: (itemId: string) => boolean;
-  toggleItemExpansion: (event: React.SyntheticEvent, itemId: string) => void;
-  expandAllSiblings: (event: React.KeyboardEvent, itemId: string) => void;
+  /**
+   * Check if an item is expanded.
+   * @param {TreeViewItemId} itemId The id of the item to check.
+   * @returns {boolean} `true` if the item is expanded, `false` otherwise.
+   */
+  isItemExpanded: (itemId: TreeViewItemId) => boolean;
+  /**
+   * Check if an item is expandable.
+   * Currently, an item is expandable if it has some children.
```

Comment:
Nitpick: the word 'some' feels weird here 🤔 
```suggestion
   * Currently, an item is expandable if it has children.
```